### PR TITLE
APPS-2410 release nextaur 1.7.1

### PR DESCRIPTION
**Pull request by YuxinShi0423**

### CODE REVIEW

**Comment by YuxinShi0423** [comment]
https://jenkins-vstg.internal.dnanexus.com/view/dx-toolkit%20release%20board/job/dx-toolkit/job/dxpy-branch-integration-tests-on-staging-20.04-python3/693/

**Review by r-i-v-a — APPROVED** [review]

**Review by mhrvol — APPROVED** [review]
LGTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +11,7 @@ Categories for each release: Added, Changed, Deprecated, Removed, Fixed, Securit
 ### Fixed
 
 * Bugfix regarding multi-assay assay selection in `dx extract_assay expression`
+* Released Nextaur 1.7.1. It contains multiple fixes in resolving docker image digests, handling file path collision and OOM when creating new input stream.
```
Comment (r-i-v-a) [inline CHANGELOG.md:14]: Changelog entry should move to unreleased section @YuxinShi0423

**Review by r-i-v-a — CHANGES_REQUESTED** [review]
Changelog entry should move to unreleased section

**Review by r-i-v-a — APPROVED** [review]
Fixed the changelog entry; please continue with the release.

**Pull request merged**